### ISSUE TITLE
When I ran the 'activations.py', I met an error

### ISSUE THREAD

**Issue by tianyu0523** [FHainzl/Visualizing_Understanding_CNN_Implementation#2]
Could you please help me solve it?![_20180812142809](https://user-images.githubusercontent.com/25603473/44001539-c142fff0-9e3c-11e8-81ba-0c76f653b10c.png)

**Comment by tianyu0523** [comment]
Or what is the 'required.txt' for this model?

**Comment by tianyu0523** [comment]
Or what is the 'requirement.txt' for this model?

**Comment by pvop** [comment]
I get the same error. Because the tensorflow.keras version is different, this code use keras.layers.merge, but there is not merge in new keras. So you should use keras.layers.Concatenate(), so the solution is:
 conv_2 = [Conv2D(128, 5, activation="relu", name='conv_2_' + str(i + 1))(split_tensor(ratio_split=2, id_split=i)(conv_2)) for i in range(2)]
conv_2 = keras.layers.Concatenate(axis=1,name="conv_2")(conv_2)
try to revise all merge.
